lib/screenshots.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Helper functions for taking screenshots with webdriver.
+ */
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import {driver, WebDriver} from './index';
+
+/**
+ * Uses driver to take a screenshot, and saves it to MW_SCREENSHOT_DIR/screenshot-{N}.png if the
+ * MW_SCREENSHOT_DIR environment variable is set.
+ *
+ * - relPath may specify a different destination filename, relative to MW_SCREENSHOT_DIR.
```

Comment:
It looks like `relPath` can specify subdirs, but I think the special `{N}` flag should only be used in the filename, is this correct? 
Maybe worth adding a mention in the doc (or a simple example) to prevent readers with a twisted mind from wondering about it.

Reply:
Good point :) will add.